core/backend/src/CustomViewState3dCreator.ts

```diff
@@ -39,9 +42,10 @@ export class CustomViewState3dCreator {
 
   private async _getAllCategories(): Promise<Id64Array> {
     // Only use categories with elements in them
+    Logger.logInfo(loggerCategory, `Starting getAllCategories query.`)
```

Comment:
nit; none of these logs need backticks for the actual message, theyre not templated

Reply:
Changed